spec/swagger_helper.rb

```diff
@@ -166,6 +166,150 @@ def load_swagger_yaml(*relative_path)
       },
       paths: load_swagger_yaml('hand_coded_paths.yaml'),
     },
+
```

Comment:
I've just realised that we do not need to change the spec,
I think we use this only for generating docs with Rswag.